HarpGuru/HarpFace/HarpCell/index.ts

```diff
@@ -1,2 +1,2 @@
 export { HarpCell } from './components'
-export type { HarpCellProps, YXCoord, Coord } from './types'
+export type { YXCoord, Coord } from './types'
```

Comment:
While the folder structure looks like it could use some improvements; ie the types could probably be defined in the same files as they are used, the folder tidyup project (https://github.com/js-jslog/harpnative/issues/39) will take place after the technical debt has been addressed (https://github.com/js-jslog/harpnative/issues/45). These hooks improvements are one move towards addressing the tech debt.